bip-0121.mediawiki

```diff
@@ -115,7 +115,7 @@ Send PoP for a transaction with label "video 42923" to
 <nowiki>https://www.example.com/pop/352</nowiki>, using nonce
 <tt>0x73 0xd5 0x1a 0xbb 0xd8 0x9c</tt>:
 <pre>
- btcpop:?p=https://www.example.com/pop/352&n=zgWTm8yH&label=video 42923
+ btcpop:?p=https://www.example.com/pop/352&n=zgWTm8yH&label=video%2042923
```

Comment:
Why is '+' a valid encoding for ' ' (space) in bitcoin: URIs? Isn't that a URL encoding thing? I think there is nothing in RFC 3986 about that, nor in BIP21? I think '+' means '+' in RFC 3986.

However '+' is interpreted as space (' ') by most wallets (that I tried), but I do think it's a mistake to do so.